uber/models.py

```diff
@@ -1094,9 +1094,11 @@ class Attendee(MagModel, TakesPaymentMixin):
     dept_checklist_items = relationship('DeptChecklistItem', backref='attendee')
 
     _repr_attr_names = ['full_name']
-    __table_args__ = (
-        UniqueConstraint('badge_num', deferrable=True, initially='DEFERRED'),
-    )
+
+    if Session.engine.dialect == 'postgresql':
```

Comment:
The main fix for all the errors in our unit tests. One line, 3+ hours to find it.

For some reason, checking SQLALCHEMY_URL never worked, even if I explicitly changed it in conftest right before initializing the DB. This is more accurate anyway, I think.